ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/statementofmeans/CourtOrderMapperTest.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.cmc.ccd.mapper.statementofmeans;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.gov.hmcts.cmc.ccd.config.CCDAdapterConfig;
+import uk.gov.hmcts.cmc.ccd.domain.statementofmeans.CCDCourtOrder;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.CourtOrder;
+
+import java.math.BigDecimal;
+
+import static uk.gov.hmcts.cmc.ccd.assertion.Assertions.assertThat;
+
+@SpringBootTest
+@ContextConfiguration(classes = CCDAdapterConfig.class)
+@RunWith(SpringJUnit4ClassRunner.class)
+public class CourtOrderMapperTest {
+
+    @Autowired
+    private CourtOrderMapper mapper;
+
+    @Test
+    public void shouldMapCourtOrderToCCD() {
+        //given
+        CourtOrder courtOrder = CourtOrder.builder()
+            .amountOwed(BigDecimal.TEN)
+            .details("Reference")
+            .monthlyInstalmentAmount(BigDecimal.ONE)
+            .build();
+
+        //when
+        CCDCourtOrder ccdCourtOrder = mapper.to(courtOrder);
+
+        //then
+        assertThat(courtOrder).isEqualTo(ccdCourtOrder);
+
+    }
+
+    @Test
+    public void shouldMapCourtOrderFromCCD() {
+        //given
+        CCDCourtOrder ccdCourtOrder = CCDCourtOrder.builder()
```

Comment:
why don't you extract it? Both objects seem to be the same, aren't they?

Reply:
one is claim-store and other is CCD model, not actually same.